### ISSUE TITLE
Feature Request: Custom status filters/workflow

### ISSUE THREAD

**Issue by A-Lawrence** [osTicket/osTicket-1.8#1733]
Hello,

Would it be possible to allow the status links at the top of the SCP (i.e. Open, Answered, Overdue, etc) to be customised, or allow the workflow of tickets to be customised?

For example, our workflow would be improved by being able to arrange tickets like so:

* New (a brand new ticket which hasn't been replied to). All tickets default to this when opened.
* Responded (client has posted a response to our reply). All tickets change to this when the client adds a new response.
* Further Action (awaiting action by the staff member). Staff would set this manually to indicate further action required. 
* Awaiting Reply (this ticket has been replied to by the staff member and a response from client is required). Default status selection when staff go to post a reply.
* Closed (resolved OR no response from client required)

**Comment by rp-tech** [comment]
+1, would like to see additional statuses as well.

On top of the statuses mentioned by A-Lawrence I would also like to see an additional status for 'Waiting on Parts / Repair' in other words the machine is out for repair under warranty depot, or else it's waiting on a part to arrive before repair can proceed further.

Thank you all very much for your amazing work! Keep it up (maybe someday I will get an opportunity to buy greezy and protich a couple of beers.)

**Comment by kordianbruck** [comment]
I think it would be easier to implement a quick filter/search option where you could select your custom status from a drop down and it would immediately filter the tickets. 

But yeah, I agree. Custom statuses make little to no sense currently because they are so hard to use (advanced filter is nice but too slow for quick access).

**Comment by greezybacon** [comment]
@kordianbruck I like the way you think. I'm lobbying for such a feature in v1.10

**Comment by mfelber** [comment]
I also think that "*a quick filter/search option where you could select your custom status from a drop down and it would immediately filter the tickets.*" would increase the acceptance/usage of the custom statues.

Today it is not very intuitive or comfortable to use them..

So I'm looking forward to see such a nice improvement of the gui in v1.10 :-)
